src/constants/blacklistConfig.js

```diff
@@ -140,6 +145,38 @@ const programs = {
 		'next_stage',
 		'created_at',
 		'updated_at',
+		'is_under_edit',
+		'deleted_at',
+		'metaData',
+		'meta',
+		'stage',
+		'organizations',
+		'is_comments',
+	],
+	resources: [
+		'user_id',
+		'organization_id',
+		'created_by',
+		'updated_by',
+		'review_type',
+		'type',
+		'blob_path',
+		'status',
+		'last_reviewed_on',
+		'published_on',
+		'submitted_on',
+		'published_id',
+		'reference_id',
+		'next_stage',
+		'created_at',
+		'updated_at',
+		'is_under_edit',
+		'deleted_at',
+		'meta',
+		'is_comments',
+		'formMeta',
```

Comment:
@priyanka-TL what is the diff between meta and fromMeta?

Reply:
@aks30  its  the keys frontend is not removed from their end hence we are removing

Reply:
But why we have 2 different keys, what data gets stored in each?